fastplotlib/layouts.py

```diff
@@ -49,6 +55,16 @@ def __init__(
         """
         self.shape = shape
 
+        positions = list()
+        for x in range(self.shape[0]):
+            for y in range(self.shape[1]):
+                positions.append((x, y))
+
+        self.indexers = list()
+        for i in range(self.shape[0] * self.shape[1]):
+            self.indexers.append(GridPlotIndexer())
+            self.indexers[i].index = positions[i]
+
```

Comment:
add just `__getitem__`

I think this should be read-only, we don't ones users messing with the subplots manually in the gridplot so `__setitem__` is not necessary (I think).